### PR TITLE
fix: some site structure changes

### PR DESCRIPTION
**Pull request by vincentaudebert**
## Type

- Enhancement

#### What is expected?

Better site structure and improvements of some sections

#### The following changes where made:

(Describe what you did)

1. Change some text

2. Adds open source section + community

3. Adds code examples

## Relevant logs and/or screenshots

Screenshot 2021-11-09 at 19 14 25

Screenshot 2021-11-09 at 19 14 36

### CODE REVIEW

**Comment by vincentaudebert** [comment]
What icons do we need:

- Icons for Introduction blocks (Accessibility, Customizable, DevXP)
- Icons for Open source libs
- Better Slack/community icon?
- Why is the illustration a man? Should we be more inclusive?

**Review by matthprost — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/15812968/141109707-44ca8e97-9f15-49f0-baab-fe1362a7c983.mov

Copy / paste button is broken on getting started 🤔

File: src/pages/home/GettingStarted.tsx [hunk 1]
```diff
@@ -23,10 +24,14 @@ const GettingStarted = (): JSX.Element => (
...
         </Typography>
         <Typography textAlign="justify" mb={5}>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
-          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-          aliquip ex ea commodo consequat.
+          Inside your react project start using Scaleway UI by running:
+        </Typography>
+        <CopyBox>
+          <CopyBox.Command title="Install" command={install} />
```
Comment (matthprost) [inline src/pages/home/GettingStarted.tsx:30]: I think we should add code snippet for npm too

**Review by matthprost — CHANGES_REQUESTED** [review]

**Comment by matthprost** [comment]
Seems like dark / light mode doesn't work anymore on this PR

File: src/pages/home/GettingStarted.tsx [hunk 2]
```diff
@@ -23,10 +31,22 @@ const GettingStarted = (): JSX.Element => (
...
+          of the following:
+        </Typography>
+        <RelativeDiv>
+          <CopyBox>
+            <CopyBox.Command title="Install" command={install} />
+          </CopyBox>
+        </RelativeDiv>
+        <RelativeDiv>
+          <CopyBox>
+            <CopyBox.Command title="Install" command={installNpm} />
```
Comment (matthprost) [inline src/pages/home/GettingStarted.tsx:44]: Can you change `CopyBox.Command` props to be able to receive `showLineNumbers` as prop and pass it to `SyntaxHighlighter` ?

Then here you could hide line number as it is just a one line command, and make it true by default

**Review by matthprost — CHANGES_REQUESTED** [review]

File: src/pages/home/index.tsx [hunk 3]
```diff
@@ -1,30 +1,73 @@
...
+      <StyledCommunityLeft>
+        <Image src={slackLogo} width="48" height="48" alt="Slack logo" />
+        <StyledTypographyContainer>
+          <Typography variant="title">Connect with the community</Typography>
+          <Typography variant="description">
+            Feel free to ask questions, report issues, and meet new people.
+          </Typography>
+        </StyledTypographyContainer>
+      </StyledCommunityLeft>
+      <Button size="large">Join the Community Slack</Button>
```
Comment (matthprost) [inline src/pages/home/index.tsx:69]: No link to the slack community ?

**Review by matthprost — CHANGES_REQUESTED** [review]

File: src/pages/home/index.tsx (hunk 3 above)
Comment (vincentaudebert) [reply src/pages/home/index.tsx:69]: Not yet, @philibea was meant to create it and give it to me. I will ask.

Comment (philibea) [reply src/pages/home/index.tsx:69]: Created @vincentaudebert

**Review by matthprost — APPROVED** [review]

File: src/components/CopyBoxCommand.tsx [hunk 4]
```diff
@@ -54,23 +67,29 @@ const CopyBox = ({ children }: CopyBoxProps): JSX.Element => {
   )
 }
 
+CopyBox.defaultProps = {
```
Comment (chambo-e) [inline src/components/CopyBoxCommand.tsx:70]: Not needed :)

You can disable the rule for tsx files if you want :)

**Pull request merged**